src/ch4_component.cpp

```diff
@@ -140,28 +141,6 @@ void CH4Component::prepareToRun() throw ( h_exception ) {
 void CH4Component::run( const double runToDate ) throw ( h_exception ) {
 	H_ASSERT( !core->inSpinup() && runToDate-oldDate == 1, "timestep must equal 1" );
 
-    // modified from Wigley et al, 2002
-    // https://doi.org/10.1175/1520-0442(2002)015%3C2690:RFDTRG%3E2.0.CO;2
```

Comment:
@mnichol3 where did all of this code go?

Reply:
was this redone win alexey's commits?

Reply:
@cahartin some of this code was marked as a conflict and removed. This section might have been a duplicate because a nearly-identical block of code is present in my [latest commit](https://github.com/mnichol3/hector/commit/1f6c32421a3f97e091c9bb82eaf479b26904726f#diff-14751e5348624ea811d1d661220b712c)